src/features/singleRight/delegate/ChooseServicePage/ChooseServicePage.tsx

```diff
@@ -30,7 +31,9 @@ export const ChooseServicePage = () => {
   const [popoverOpen, setPopoverOpen] = useState(false);
   const buttonRef = useRef<HTMLButtonElement | null>(null);
   const delegableChosenServices = useAppSelector((state) =>
-    state.singleRightsSlice.servicesWithStatus.filter((s) => s.status !== 'NotDelegable'),
+    state.singleRightsSlice.servicesWithStatus.filter(
```

Comment:
Weird with + symbol for try again

<img width="193" alt="image" src="https://github.com/Altinn/altinn-access-management-frontend/assets/31471142/ae34f493-a791-49a9-9eb1-ce28914da982">

Reply:
We'll review it in design meeting next week